packages/ChangesReloaded-Core.package/CRChangeListItem.class/instance/hash.st

```diff
@@ -0,0 +1,9 @@
+comparing
+hash
+
+	| hash |
+	hash := self changeRecord hash.
+	hash := self description hashWithInitialHash: hash.
+	hash bitXor: self isPicked asBit.
+	hash := self nameOfAffectedClass hashWithInitialHash: hash.
```

Comment:
Right now, yes. What if we change how the description is built?
I'd rather argue to not include the description

Reply:
Hang on, so hash should deliver the same value if the obejcts differ only at the Object Identity? This is not the case right now

Reply:
As long as we deterministically build the `description` and `nameOfAffectedClass` based on the `changeRecord` and `isPicked`, just using those two hashes directly is fine.
If we change how the description is built, then we usually change that for all `CRChangeListItem` instances, so their hash is still the same.

Reply:
Felix says we should only use the properties for hashing that we also use for = (since objects that are equal should have the same hash) which makes a lot of sense for grit and me